### DISCUSSION TITLE
localStorage

### DISCUSSION THREAD

**Discussion by chdelfosse** [epeios-q37/atlas-python#13] [Ideas]
The "core" concept as shown in the examples is fundamental in supporting several users,
I think that having access to `HTML5`'s `localStorage` would enable each one of them to track their progress.

I must add that I have a working (fallback) solution.

**Comment by epeios-q37** [comment]
I did some tests, and It would be easy to give access to both *localStorage* and *sessionStorage*. But, there is a big  problem. All the web applications from the same domain (here *faas.q37.info*) shares the same storage. That is,  one application can have access and also modify the items used by the other applications using the same server. This is a serious security issue.
So, simply giving access to storage in the same way as with *JavaScript* is too hazardous. I have to find something equivalent, but more secure.

**Comment by chdelfosse** [comment]
I understand. I will use cookies instead and I will remove the three `localStorage` elements that I had misguidedly inserted into the site.
